### DISCUSSION TITLE
How are EANs/identifiers supposed to work in SearchCatalogItems?

### DISCUSSION THREAD

**Discussion by callumb123** [amzn/selling-partner-api-models#3528] [General]
I have an open support case with this issue but after 3 weeks with no response I am hoping someone here might know the answer!

I am using the SearchCatalogItems endpoint (UK) to search using product EANs and UPCs. The results I am getting are quite confusing. Could anyone help me understand the reason I am seeing the following 2 scenarios:

Case 1:
Using a single EAN "5702016888720". 
I am getting back two products which both have 2 EAN identifiers and both have this "5702016888720" EAN in their identifiers - B08G4KZPG4 & B08KVYS3R7. Only one of these ASINs is correct (B08G4KZPG4) the other one refers to a completely different product. 
How am I supposed to know which one is correct and why is this identifier on a different product?

Case 2:
Using a single EAN "5060164140108". 
I am getting back 1 product which has 18 individual identifiers (EANs, GTINs and UPCs). Most of the identifiers don't seem to be valid (checked using external barcode lookup) and the others refer to completely different products. 
I understand some products will have more than one barcode referring to the same product, however this is not the case here and instead the identifiers are bloated with a lot of false information. Anyone know why this is and how we are supposed to confirm which identifiers are correct?

**Comment by naveedali8086** [comment]
@callumb123  or anyone, could you guys explain little bit the difference between 
[**Catalog Items API**](https://developer-docs.amazon.com/sp-api/docs/catalog-items-api-v2022-04-01-reference) and [**Listings Items API**](https://developer-docs.amazon.com/sp-api/docs/listings-items-api-v2021-08-01-reference) or which should be used in what case.

May question seems very basic but I read the docs of both, still not clear the use case of both the APIs.

**Reply by callumb123** [reply]
@naveedali8086 It would be better to raise a separate discussion thread as this isn't really related to my original post.

**Reply by sungolivia** [reply]
agreed with what @callumb123  said but just to provide quick reads, have you tried reading their use case guide?
If you can find which use case fits your needs, maybe it will help! But Catalog is mainly for searching for an ITEM in Amazon catalog where as Listing is if you want to do operations for specific Amazon Listing. 

https://developer-docs.amazon.com/sp-api/docs/catalog-items-api-v2022-04-01-use-case-guide
https://developer-docs.amazon.com/sp-api/docs/listings-items-api-v2021-08-01-use-case-guide
